src/Sarif.UnitTests/SarifCurrentToVersionOneTests.cs

```diff
@@ -198,5 +185,208 @@ public void SarifTransformerTests_ToVersionOne_OneRunWithLogicalLocations()
 
             VerifyCurrentToVersionOneTransformation(V2LogText, V1LogExpectedText);
         }
+
+        [Fact]
+        public void SarifTransformerTests_ToVersionOne_OneRunWithFiles()
+        {
+            const string V2LogText =
+@"{
+  ""$schema"": ""http://json.schemastore.org/sarif-2.0.0"",
+  ""version"": ""2.0.0"",
+  ""runs"": [
+    {
+      ""tool"": {
+        ""name"": ""CodeScanner"",
+        ""semanticVersion"": ""2.1.0""
+      },
+      ""invocations"": [
```

Comment:
This isn't valid SARIF. If `invocations` exists, it is an array of one or more unique `invocation` objects. `null` is not a valid array entry.

SARIF does allow an empty array (see §3.6.1) in the same way it allows an empty object (§3.5), so you can do whichever you like here.

Reply:
Yes I fixed this in the upcoming commit